docs_user/modules/proc_deploying-a-ceph-ingress-daemon.adoc

```diff
@@ -2,16 +2,21 @@
 
 = Deploying a {Ceph} ingress daemon
 
-To match the cardinality diagram, you use cephadm labels to refer to a group of nodes where a given daemon type should be deployed. For more information about the cardinality diagram, see xref:ceph-daemon-cardinality_migrating-ceph[{Ceph} daemon cardinality].
-`HAProxy` is managed by {OpenStackPreviousInstaller} through `Pacemaker`: the three running instances at this point will point to the old RGW backends, resulting in a broken configuration.
+To ensure that you distribute the correct number of services to your available {Ceph} nodes, you use `cephadm` labels to refer to a group of nodes where a given daemon type is deployed. For more information about the cardinality diagram, see xref:ceph-daemon-cardinality_migrating-ceph[{Ceph} daemon cardinality].
+
+`HAProxy` is managed by {OpenStackPreviousInstaller} through Pacemaker. The three running instances are pointing to the old RGW back ends, resulting in a broken configuration.
+
 ifeval::["{build}" != "upstream"]
-Since you are going to deploy the Ceph ingress daemon, the first thing to do is remove the existing `ceph_rgw` config, clean up the config created by {OpenStackPreviousInstaller} and restart the service to make sure other services are not affected by this change.
+To deploy the Ceph ingress daemon, you perform the following actions:
+
+. Remove the existing `ceph_rgw` configuration.
+. Clean up the configuration created by {OpenStackPreviousInstaller}.
+. Restart the service to ensure that other services are not affected by this change.
 endif::[]
 ifeval::["{build}" != "downstream"]
```

Comment:
I think this same sentence we have downstream can be removed. We can keep a single version for both upstream and downstream, and reduce the scope of the if statement to the downstream vs upstream variables.
@klgill wdyt?